pygments/lexers/ul4.py

```diff
@@ -0,0 +1,261 @@
+"""
+    pygments.lexers.ul4
+    ~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the UL4 templating language.
+
+    More information: https://python.livinglogic.de/UL4.html
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments import lexer, token
+from pygments.lexers import web, python
+
+__all__ = [
+    'UL4Lexer',
+    'HTMLUL4Lexer',
+    'XMLUL4Lexer',
+    'CSSUL4Lexer',
+    'JavascriptUL4Lexer',
+    'PythonUL4Lexer',
+]
+
+
+class UL4Lexer(lexer.RegexLexer):
+    """
+    Generic lexer for UL4.
+
+    .. versionadded:: 2.12
+    """
+
+    flags = re.MULTILINE | re.DOTALL | re.UNICODE
```

Comment:
No need for `re.UNICODE`, see #2058.

Reply:
Fixed in https://github.com/doerwalter/pygments/commit/8ce6588cc5644097f247d2156cf388d24550034b